docs/pages/connect-your-client/introduction.mdx

```diff
@@ -310,8 +310,8 @@ either directly or through proxy tunnels.
   Learn more about it from [Using the tsh Command Line Tool](./tsh.mdx).
 
 - Teleport Connect is a graphic utility for connecting to resources through
-Teleport. You can initiate Server Access, connect to databases, and start
-Desktop Access sessions. See [Using Teleport Connect](./teleport-connect.mdx).
+  Teleport. You can connect to servers, databases, and desktops. See [Using
```

Comment:
I don't know how this got here, but it's incorrect. You can not connect to desktops with Teleport Connect - only servers, databases, and Kube clusters.